aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -0,0 +1,146 @@
+---
+title: HttpSys web server implementation in ASP.NET Core
+author: rick-anderson
+description: Introduces HttpSys, a web server for ASP.NET Core on Windows. Built on the Http.Sys kernel mode driver, HttpSys is an alternative to Kestrel that can be used for direct connection to the Internet without IIS.
+keywords: ASP.NET Core, HttpSys, HttpListener, url prefixes, SSL 
+ms.author: riande
+manager: wpickett
+ms.date: 08/07/2017
+ms.topic: article
+ms.assetid: 0a7286e4-6428-424e-b5c4-5c98815cf61c
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/servers/HttpSys
+---
+# HttpSys web server implementation in ASP.NET Core
+
+By [Tom Dykstra](http://github.com/tdykstra) and [Chris Ross](https://github.com/Tratcher)
+
+> [!NOTE]
+> This topic applies only to ASP.NET Core 2.0 and later. In earlier versions of ASP.NET Core, HttpSys is named [WebListener](WebListener.md).
+
+HttpSys is a [web server for ASP.NET Core](index.md) that runs only on Windows. It's built on the [Http.Sys kernel mode driver](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx). HttpSys is an alternative to [Kestrel](kestrel.md) that offers some features that Kestel doesn't. **HttpSys can't be used with IIS or IIS Express, as it isn't compatible with the [ASP.NET Core Module](aspnet-core-module.md).**
+
+HttpSys supports the following features:
+
+- [Windows Authentication](xref:security/authentication/windowsauth)
```

Comment:
> Does this list mean that you all agreed on - for 1st level bullets?

No it means we didn't see the need to require one or the other -- the both work, but the norm here is asterisks.